File: twinprimes-python/twinprimes.py
#!/usr/bin/python

def printTwinPrimesToN(n):
	arrayOfPrimes = list()
	arrayOfPrimes.append(2)
	arrayOfPrimes.append(3)
	
	for i in range(4,n):
		valueIsPrime = True
		for j in range(0,len(arrayOfPrimes)):
			if(valueIsPrime and i % arrayOfPrimes[j] == 0):
				valueIsPrime = False
		if(valueIsPrime):
			arrayOfPrimes.append(i)
			if arrayOfPrimes[len(arrayOfPrimes) - 1] - arrayOfPrimes[(len(arrayOfPrimes) - 2)] == 2:
				print "<",arrayOfPrimes[(len(arrayOfPrimes) - 2)],",",arrayOfPrimes[(len(arrayOfPrimes) - 1)],">"
	return
	
printTwinPrimesToN(100000)

File: twinprimes-nodejs/twinprimes.js

function printTwinPrimesToN(n){
	var arrayOfPrimes = [2];
	for(i = 3; i < n; i++) {
		valueIsPrime = true;
		
		for (j = 0; j < arrayOfPrimes.length; j++){
			if(valueIsPrime && i % arrayOfPrimes[j] == 0) {
				valueIsPrime = false;
			}
		}
		
		if(valueIsPrime == true) {
			arrayOfPrimes.push(i);
			if(arrayOfPrimes[arrayOfPrimes.length - 1] - arrayOfPrimes[arrayOfPrimes.length - 2] == 2){
				console.log("<".concat(arrayOfPrimes[arrayOfPrimes.length - 2],", ",arrayOfPrimes[arrayOfPrimes.length - 1],">"));
			}
		}
	}
}

printTwinPrimesToN(100000)